edgelet/iotedged/src/error.rs

```diff
@@ -144,6 +144,7 @@ impl From<&ErrorKind> for i32 {
             ErrorKind::Initialize(InitializeErrorReason::InvalidDeviceConfig) => 150,
```

Comment:
While you're here, could you fix the typo on line 139 in the URL please? (`https://www.freebsd.org/cgCould not get module i/man.cgi` -> `https://www.freebsd.org/cgi/man.cgi`)

Reply:
Done.